crates/polars-plan/src/client/check.rs

```diff
@@ -3,8 +3,18 @@ use polars_core::error::{PolarsResult, polars_err};
 use crate::constants::POLARS_PLACEHOLDER;
 use crate::dsl::{DslPlan, FileScanDsl, ScanSources, SinkType};
 
+#[cfg_attr(feature = "python", pyo3::pyclass(eq, eq_int))]
+#[derive(PartialEq, Clone)]
+pub enum LocalScansPolicy {
```

Comment:
Given that we would need to break DSL anyway if we update. Let's make it a boolean flag for now as we only have two states.